src/pybind/mgr/restful/api/pool.py

```diff
@@ -34,7 +34,11 @@ def patch(self, **kwargs):
         """
         Modify the information for the pool id
         """
-        args = request.json
+	try:
+            args = request.json
+        except ValueError:
+            response.status = 400
+            return {'message': 'Bad request'}
```

Comment:
Can we be a bit more specific here and return something like 'Bad request: malformed json'?